fasta/fasta.go

```diff
@@ -27,14 +27,24 @@ type FastaMap map[string][]dna.Base
 // Read in a fasta file to a []Fasta struct. All sequence records must be preceded by a name line starting with '>'.
 // Each record must have a unique sequence name
 func Read(filename string) []Fasta {
+	return read(filename, NextFasta)
+}
+
+// ReadForced functions identically to Read, but any invalid characters in the sequence will be masked to N.
+func ReadForced(filename string) []Fasta {
+	return read(filename, NextFastaForced)
+}
+
+// read is a helper function for Read and ReadForced to reduce duplication
+func read(filename string, nextFunc func(*fileio.EasyReader) (Fasta, bool)) []Fasta {
```

Comment:
aahhhh, that is an interesting way to do it.  I was picturing passing a bool down through the functions, but I think this is probably a more elegant way to do it.

Reply:
I figured since it we will not want to force the conversion 99.9% of the time, it would be less disruptive to just make a new function rather than adding a bool input to fasta.Read.

Reply:
Yes, I understand.  I was thinking of something like:
fasta.Read(filename) calling: readHelper(filename, true)
and
fasta.ReadForced(filename) calling: readHelper(filename, false)
but I think your solution is more elegant since you don't need to interpret the bool inside the function